tests/integration/actions.go

```diff
@@ -772,6 +772,7 @@ func (tr TestRun) relayPackets(
 	if err != nil {
 		log.Fatal(err, "\n", string(bz))
 	}
+	time.Sleep(10 * time.Second)
```

Comment:
@MSalopek comments here would be nice, but shouldn't block merging

Reply:
Atm this has a merge conflict.
We should retain the changes from main - those have some comments and actually wait for blocks to be produced instead of sleeping.

Reply:
Sounds good, merged main 👍